soot-infoflow-summaries/summariesManual/java.util.ListIterator.xml

```diff
@@ -36,7 +36,11 @@
 			</flows>
 		</method>
 		<method id="void set(java.lang.Object)">
-			<flows>
+			<clears>
+				<clear sourceSinkType="Field" AccessPath="[java.util.Iterator: java.lang.Object[] innerArray]"
```

Comment:
Ein Element in einem Iterator zu ändern darf nicht die Taints von allen anderen Elementen in der zugrundeliegenden Collection entfernen.